grpc/dataplane_grpc.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package config;
+
+option go_package = "dataplane/grpc";
+
+service ConfigService {
+  rpc GetConfig (GetConfigRequest) returns (GlobalConfig);
+  rpc GetConfigGeneration (GetConfigGenerationRequest) returns (GetConfigGenerationResponse);
+  rpc UpdateConfig (UpdateConfigRequest) returns (UpdateConfigResponse);
+}
+
+message GetConfigRequest {}
+
+message UpdateConfigRequest {
```

Comment:
It may be convenient to have some form of request Id ?

Or, are we relying on the generation within the config?

If that's the case, I'd still consider adding something here, because if we need other types of requests, that would be common to all of them?

Reply:
There couldn't be other types of update config requests - it's a very specific one. Generation uniquely identifies the content meaning that if content changed anyhow generation will be incremented as well.